### DISCUSSION TITLE
Alias de commandes et restriction par statuts (Vip, Modo, Sub)

### DISCUSSION THREAD

**Discussion by Klelian** [skarab42/marv#28] [Ideas]
Pour les commandes:
- une fonction alias de commande qui permettrait de lancer la même commande sous plusieurs "!...." différents
- la possibilité de désigner qui peut utiliser une commande "VIP / Folowers / Sub / Utilisateur ciblé"

**Comment by skarab42** [comment]
- Pour les alias je suis pas sur de comprendre l'intérêt.
- La gestion des statuts des utilisateurs est prévus dans une v1.x.

P.S: Merci de faire une discussion par idée avec un titre explicite, comme ça je peu les transférer plus simplement en issues puis en projet.

**Comment by skarab42** [comment]
Pour la gestion de status c'est good depuis la v1.7.0
